tools/model_converters/vitjax2mmseg.py

```diff
@@ -0,0 +1,128 @@
+import argparse
```

Comment:
Credit?

Reply:
Please also remove unnecessary comments.

Reply:
Credit to @rstrudel .